ocw-course-v2/ocw-studio.yaml

```
@@ -109,6 +109,67 @@ collections:
           filter_type: "equals"
           value: "Video"
 
+  - category: Content
+    name: link
+    label: Links
+    folder: content/links
+    fields:
+      - label: Title
+        name: title
+        widget: string
+        required: true
+
+      - label: Text
+        name: text
+        widget: string
+        required: true
+        help: Text to display as the link.
+
+      - label: URL
+        name: external_url
+        widget: string
+        required: true
+        help: The URL. For example, https://ocw.mit.edu/
```

This is a confusing example. 

```suggestion
        help: The URL. For example, https://raise.mit.edu/ or https://owl.english.purdue.edu/owl/resource/747/01/ 
```